reconcile/test/test_utils_state.py

```diff
@@ -0,0 +1,65 @@
+import os
+from tempfile import NamedTemporaryFile
+
+import pytest
+import boto3
+from moto import mock_s3
+from reconcile.utils.state import State
+
+
+@pytest.fixture
+def accounts():
+    """Account name is needed to instantiate a State"""
+    return [{'name': 'some-account'}]
+
+
+@pytest.fixture
+def aws_credentials():
+    """Mocked AWS Credentials for moto."""
+    os.environ['AWS_ACCESS_KEY_ID'] = 'testing'
+    os.environ['AWS_SECRET_ACCESS_KEY'] = 'testing'
+    os.environ['AWS_SECURITY_TOKEN'] = 'testing'
+    os.environ['AWS_SESSION_TOKEN'] = 'testing'
+
+
+@mock_s3
+def test_ls(accounts, aws_credentials, mocker):
```

Comment:
While you're in here, worth adding one more quick test for where the contents can be empty?

Reply:
Done.